espnet/tts/pytorch_backend/tts.py

```diff
@@ -397,16 +395,18 @@ def decode(args):
         logging.info('args: ' + key + ': ' + str(vars(args)[key]))
 
     # define model
-    tacotron2 = Tacotron2(idim, odim, train_args)
+    model_class = dynamic_import(train_args.model_module)
```

Comment:
assert isinstance(model_class, TTSInterface)

Reply:
BTW, how about split `train` and `decode` into separated files e.g., `tts_train.py`, `tts_decode.py`. I think  the function`decode` does not depend on many of `espnet.tts.pytorch_backend_tts.py`

Reply:
Sorry, I could not catch up your meaning.
`tts_train.py` and `tts_decode.py` are already separated.

Reply:
Wait, model_class is not instance, so this `assert isinstance(model_class, TTSInterface)` is failed.
We do not have to check it.

Reply:
> 
> 
> Sorry, I could not catch up your meaning.
> `tts_train.py` and `tts_decode.py` are already separated.

Sorry. I mean split `espnet/tts/pytorch_backend/tts.py` into `espnet/tts/pytorch_backend/tts_{train, decode}.py` not `espnet/bin/tts_{train, decode}.py`

Reply:
> Wait, model_class is not instance, so this `assert isinstance(model_class, TTSInterface)` is failed.
> We do not have to check it.

You are right. Please assert this very next line.
```python
model_class = dynamic_import(args.model_module)
model = model_class(idim, odim, args)
assert isinstance(model, TTSInterface)
```

Reply:
@kamo-naoyuki told me to use `issubclass` instead of `isinstance`!